Minor documentation updates

vparpoil opened the pull request:
- Generation of settings in the UI for new apps
- MAIL_URL env variable
- Sorting of the apps by name

vparpoil committed:
add some documentation about the related application package

and clarify the two installation processes

vparpoil committed:
sort apps by name

vparpoil committed:
adding MAIL_URL env var to mup placeholder

mail_url is necessary to send email (eg invitation emails)

vparpoil committed:
adding generation of settings in the UI when creating an app

lmachens commented on README.md:
```diff
@@ -24,7 +25,9 @@ docker run -d --name meteor-apm-server \
-### Setup steps
+### Setup steps using MUP
+
+Keep in mind when using MUP: [your user account on the server must be sudoer without password](http://meteor-up.com/docs.html#ssh-based-authentication-with-sudo)
```
Why is this necessary? It works for me without.

vparpoil replied:
the links give the details about this. maybe your username is root ?

lmachens replied:
no, but doesn't matter. this comment won't hurt :)

lmachens commented on mup-placeholder.js:
```diff
@@ -20,7 +20,8 @@ module.exports = {
       MONGO_URL: 'mongodb://mongodb/meteor',
-      MONGO_OPLOG_URL: 'mongodb://mongodb/local'
+      MONGO_OPLOG_URL: 'mongodb://mongodb/local',
+      MAIL_URL: ''
```
Do you know if mailing works?

vparpoil replied:
Yes, mailing works (I tested invinting users to an application)

lmachens commented on client/components/app/partials/noData.html:
```diff
@@ -4,40 +4,15 @@
 
     <p>2. {{{i18n 'app.config_connect_code'}}}</p>
     <div id="no-data-code-wrap">
-      <ul class="nav nav-tabs" role="tablist">
```
Actually I like to have the option to set the apm settings via code or env. So I would not remove these tabs here and better add the apm engine url to the other tabs too:
Code:
`Kadira.connect('{{state$appId}}', '{{state$secret}}', { endpoint:  apmEngineUrl });`
For env:
KADIRA_OPTIONS_ENDPOINT (https://github.com/lmachens/kadira/blob/master/lib/environment_variables.js#L67)

lmachens reviewed (changes requested):
Thank you for your pr! 
I added some comments

vparpoil committed:
adding more documentation on options to connect external apps to the APM

https://github.com/lmachens/meteor-apm-server/pull/35

vparpoil committed:
fixing indentation issues

vparpoil commented:
I just updated with the two other options to connect an app, thanks for your feedback, and sorry for the bad indentation on commit 023d8c7, I fixed it

lmachens reviewed (approved):
thx! I will add prettier soon

lmachens commented:
I added `apmEngineUrl` to the settings dialog too. 
see https://github.com/lmachens/meteor-apm-server/commit/e02f924bd27381b773798ff18824d93cd1a2532f
